release_notes/v1.5.5.md

```diff
@@ -0,0 +1,49 @@
+v1.5.4 Release Notes - July 8, 2022
+===================================
+
+v1.5.4 is a maintenance release, providing updates for the following issues in the Fabric CA: 
+
+- Closes [Issue #305](https://github.com/hyperledger/fabric-ca/issues/305) : Issue with re-enrolling certificates 
+
+- Closes [Issue #309](https://github.com/hyperledger/fabric-ca/issues/309) : Interoperability tests are failing with latest CA updates
+
+
+Dependencies
+------------
+
+Fabric CA v1.5.5 has been tested with the following dependencies:
+- Go 1.18.2
+- Alpine 3.16 (for Docker images)
+
+Changes, Known Issues, and Workarounds
+--------------------------------------
+
+This maintenance release addresses a regression in the Node and Java client SDKs, reverting the updates committed in [PR #304](https://github.com/hyperledger/fabric-ca/pull/304):
+
+- **Re-Opens** [Issue #303](https://github.com/hyperledger/fabric-ca/issues/303) : Idemix MSP Folder Structure incompatible with what Fabric expects
```

Comment:
As above, these were never released so I don't think they need to be mentioned in release notes.